### PR TITLE
Update host metrics docs and refactor files

### LINKED ISSUE #3025: [REQUEST]: Observability Infrastructure UI : Host Metrics Documentation Update
### Description

We want to update our documentation to have a single place where all host-related metrics are listed. This means we'll need to update the existing [host metrics documentation](https://www.elastic.co/guide/en/observability/current/host-metrics.html) and also update the [analyzing hosts - metrics](https://www.elastic.co/guide/en/observability/current/analyze-hosts.html#analyze-hosts-fields-metrics) to point to this master list. This should de-duplicate documentation and make everything a bit simpler for the user and for us to maintain.

> **Note** : We have consolidated the host metrics in the product to be consistent too, this isn't just a documentation change

### 1) https://www.elastic.co/guide/en/observability/current/host-metrics.html

> **Note** : Please use similar format to what you're using now in the docs (i.e. table)

See [Host-related Metrics Definitions](https://docs.google.com/document/d/1xrLMnT-Nz5TqkPT5PxNjJNLjrU4qXMsbyTWkmqLhtLw/edit#heading=h.puqzyze4cv6):

_Example - [CPU Usage](https://docs.google.com/document/d/1xrLMnT-Nz5TqkPT5PxNjJNLjrU4qXMsbyTWkmqLhtLw/edit#heading=h.2jgsb5y32qxn)_
Image

_Example Documentation_

### CPU Usage (%)
Percentage of CPU time spent in states other than Idle and IOWait, normalized by the number of CPU cores. This includes both time spent on user space and kernel space.  

100% means all CPUs of the host are busy. 

**Field Calculation** : `(average(system.cpu.user.pct) + average(system.cpu.system.pct)) / max(system.cpu.cores)`

> **Note** : We're going to be linking directly from the feature itself into this documentation to help users learn more about the metrics themselves:
>
> image

### 2) https://www.elastic.co/guide/en/observability/current/analyze-hosts.html#analyze-hosts-fields-metrics

Please update https://www.elastic.co/guide/en/observability/current/analyze-hosts.html#analyze-hosts-fields-metrics to refer to https://www.elastic.co/guide/en/observability/current/host-metrics.html (the metrics will use the same names, descriptions and definitions).

_Hopefully this will save time maintaining a single definition of metrics (instead of in multiple places)_

### Resources

[Hosts Page documentation](https://wiki.elastic.co/display/PM/Hosts+List)

### Collaboration

TBD. The docs and product team will work together to determine the best path forward.

### Point of contact.

**Main contact:** @roshan-elastic 

**Stakeholders:** @neptunian @smith

### PR DESCRIPTION
**Pull request by dedemorton**
Closes #3025 plus makes a bunch of other improvements to improve the topic flow and update some outdated content.

A full description of the changes is available in [this gdoc](https://docs.google.com/document/d/1zKjcfR-LQfZ-__dfPOsS3tNZPWlcAcgH2SAP_KvZQzM/edit?usp=sharing).

This PR includes a lot of structural changes, so it's probably best to review the rendered content in the preview rather than relying solely on the diff in GitHub.

Links to preview:
* [Infrastructure monitoring](https://observability-docs_3083.docs-preview.app.elstc.co/guide/en/observability/master/analyze-metrics.html)
* [HTML diff](https://observability-docs_3083.docs-preview.app.elstc.co/diff)

In addition to reviewing this content for accuracy, please respond to the questions I've flagged in the review (respond on the Files changed tab please):

- [x] [Do we want to publish existing docs about non-host metrics?](https://github.com/elastic/observability-docs/pull/3083/files#r1269808491) Same question applies to the sections about Kubernetes and Docker metrics.
- [x]  [Indicate whether you have any concerns about alphabetizing the metrics reference docs](https://github.com/elastic/observability-docs/pull/3083/files#r1269809054).
- [x] [Is the Infrastructure app reference still accurate? Do we want to point to it?](https://github.com/elastic/observability-docs/pull/3083/files#r1269811170)
- [x] [Should we continue to list all the source fields exposed in the Processes tab (and other places)?](https://github.com/elastic/observability-docs/pull/3083/files#r1269812112)

### CODE REVIEW

File: docs/en/observability/monitor-infra/aws-metrics.asciidoc [hunk 1]
```diff
@@ -1,6 +1,11 @@
 [[aws-metrics]]
 = AWS metrics
 
+//TODO: Rewrite this to match new format. Requires input from dev team.
+
+//REVIEWERS: This was taken from the existing docs and moved here. Do we want
```
Comment (dedemorton) [inline docs/en/observability/monitor-infra/aws-metrics.asciidoc:6]: Reviewers: Please respond to this question. What do we want to do with the existing docs about non-host metrics? Do we want to try to update them, or just leave as-is for now?

File: docs/en/observability/monitor-infra/host-metrics.asciidoc [hunk 2]
```diff
@@ -0,0 +1,218 @@
...
+
+* <<key-metrics-hosts,Hosts>>
+* <<key-metrics-cpu,CPU usage>>
+* <<key-metrics-memory,Memory>>
+* <<key-metrics-log,Log>>
+* <<key-metrics-network,Network>>
+* <<key-metrics-network,Disk>>
+
+
+//REVIEWERS: Note that the metrics are sorted alphabetically in the tables. This might
```
Comment (dedemorton) [inline docs/en/observability/monitor-infra/host-metrics.asciidoc:14]: Reviewers: Please note this comment and let me know if you have any concerns.

File: docs/en/observability/monitor-infra/kubernetes-pod-metrics.asciidoc [hunk 3]
```diff
@@ -0,0 +1,28 @@
+[[kubernetes-pod-metrics]]
+= Kubernetes pod metrics
+
+//TODO: Rewrite this to match new format. Requires input from dev team.
+
+//REVIEWERS: This was taken from the existing docs and moved here. Do we want
```
Comment (dedemorton) [inline docs/en/observability/monitor-infra/kubernetes-pod-metrics.asciidoc:6]: Reviewers: Please respond to this question about pod metrics. What do we want to do with the existing docs about non-host metrics? Do we want to try to update them, or just leave as-is for now?

File: docs/en/observability/monitor-infra/kubernetes-pod-metrics.asciidoc [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+
+| *Memory Usage* | Average of `kubernetes.pod.memory.usage.node.pct`.
+
+| *Inbound Traffic* | Derivative of the maximum of `kubernetes.pod.network.rx.bytes` scaled to a 1 second rate.
+
+| *Outbound Traffic* | Derivative of the maximum of `kubernetes.pod.network.tx.bytes` scaled to a 1 second rate.
+
+|=== 
+
+//REVIEWERS: Is the topic we are pointing to here accurate? It seems like it
```
Comment (dedemorton) [inline docs/en/observability/monitor-infra/kubernetes-pod-metrics.asciidoc:23]: Reviewers: Please respond to this question about the Infrastructure app field reference. If that topic needs updates, we'll need to do it in a follow-on PR. The main question now is whether we should like to it.

File: docs/en/observability/monitor-infra/view-infrastructure-metrics.asciidoc [hunk 5]
```diff
@@ -30,7 +34,166 @@ Alternatively, you can change this map view to a table view.
...
+====
+
+[%collapsible]
+.*Processes*
+====
+
+[role="screenshot"]
+image::images/processes-overlay.png[Host processes]
+
+//REVIEWERS: Lising all the source fields here feels like it could incur a lot
```
Comment (dedemorton) [inline docs/en/observability/monitor-infra/view-infrastructure-metrics.asciidoc:91]: Reviewers: Please respond to this question about whether we should continue to list all these source fields.

File: docs/en/observability/monitor-infra/analyze-metrics.asciidoc [hunk 6]
```diff
@@ -1,14 +1,14 @@
...
+metrics to help diagnose problematic spikes, identify high resource utilization,
+automatically discover and track pods, and unify your metrics with logs and APM
+data in {es}.
 
-Using {agent} integrations, you can ingest and analyze metrics from servers, Docker containers,
-Kubernetes orchestrations, explore and analyze application
-telemetries, and many more.
+Using {agent} integrations, you can ingest and analyze metrics from servers,
+Docker containers, Kubernetes orchestrations, explore and analyze application
+telemetries, and more.
```
Comment (smith) [inline docs/en/observability/monitor-infra/analyze-metrics.asciidoc:11]:
```suggestion
telemetry, and more.
```

File: docs/en/observability/monitor-infra/host-metrics.asciidoc (hunk 2 above)
Comment (smith) [reply docs/en/observability/monitor-infra/host-metrics.asciidoc:14]: +1 for alphabetical

**Review by roshan-elastic — APPROVED** [review]
Wow - brilliant. Thanks @dedemorton

File: docs/en/observability/monitor-infra/host-metrics.asciidoc [hunk 7]
```diff
@@ -0,0 +1,218 @@
...
+//not be ideal for some metrics that are closely related, but it's the best
+//organizing principle for reference content because it's easier to scan and
+//maintain over time. Other organizing principles fall apart over time as people
+//make updates to add/remove content. If you think related metrics need to be
+//discussed next to each other, I can break the section down into more categories.
+
```
Comment (dedemorton) [inline docs/en/observability/monitor-infra/host-metrics.asciidoc:20]:
```suggestion
```

**Comment by dedemorton** [comment]
@roshan-elastic Can you respond to the questions listed above? Thanks!

File: docs/en/observability/monitor-infra/aws-metrics.asciidoc (hunk 1 above)
Comment (roshan-elastic) [reply docs/en/observability/monitor-infra/aws-metrics.asciidoc:6]: Hey @dedemorton, let's leave them as they are. I believe they are still accurate.

File: docs/en/observability/monitor-infra/kubernetes-pod-metrics.asciidoc (hunk 4 above)
Comment (roshan-elastic) [reply docs/en/observability/monitor-infra/kubernetes-pod-metrics.asciidoc:23]: Hey @dedemorton - yes, that's still accurate. OK to publish

File: docs/en/observability/monitor-infra/host-metrics.asciidoc (hunk 2 above)
Comment (roshan-elastic) [reply docs/en/observability/monitor-infra/host-metrics.asciidoc]: Thanks for chiming in @smith

**Pull request merged**